spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -24,57 +18,24 @@
     pub.save # to update the pub.pub_hash
     pub
   end
-  let(:valid_data_for_post) do
+  let(:base_data) do
     {
-      sul_pub_id: publication.id,
       featured: false,
       status: 'denied',
       visibility: 'private',
     }
```

Comment:
Looks like `base_data` is contribution attributes now; maybe naming as such is more meaningful?